### PR TITLE
Add: Bug Issue Template

### LINKED ISSUE #15: Bug Issue Template
**Description:**
As this is a new repository, we need a bug issue GitHub template. The template should have a bug issue alone and should have the below-mentioned input fields

**Bug Issue template Requirement:**

1. Issue title should start with - [BUG]<bug title>
2. Description of the bug <Required>
3. Reproduction URL <OPTIONAL>
4. Frontend/ backend issue <Required>
5. Screenshot of the issue if UI <OPTIONAL>
6. Are you taking this issue fix <Required>

The template should be under `.github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml`

Reference: https://github.com/stevemao/github-issue-templates

**PR Requirement:**
1. Please check the above requirements are met
2. Always create a new branch on the forked repo from the master to avoid an error while merging the fix
3. Please Star this repo while forking as that would increase this repo's visibility

Cheers 🎉

### PR DESCRIPTION
**Pull request by biohazard2117**
## PR Title

The purpose of this Pull Request is to fix #15

## Description

Added the Bug Issue Template in .yml format as requested

##  Checklist
- [x] I have Made this contribution as per the CONTRIBUTING guide in this repo
- [x] I have tested in local Environment.
- [x] I have made the fix as per issue conversation
- [x] I have starred the repository ⭐

### CODE REVIEW

**Comment by biohazard2117** [comment]
@jsvigneshkanna I have added the **bug issue template**. Please review.

**Comment by jsvigneshkanna** [comment]
@biohazard2117 ,
I change required is to add hypen wherever we have space id ID attribute because that would break the workflow
Otherwise its looks better

**Comment by biohazard2117** [comment]
Can you please elaborate. I am unable to understand where you want me to make the changes and what changes.

File: .github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Bug Issue Template
+description: File a Bug Issue Report
+title: "[Bug Issue]: "
+body:
+  - type: input
+    id: bug title
```
Comment (jsvigneshkanna) [inline .github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml:6]: replace the whitespace with hypen

File: .github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      required: true
+  - type: input 
+    id: url
+    attributes:
+      label: Reporduction URL of bug 
+      placeholder: URL
+    validations:
+      required: false
+  - type: dropdown
+    id: issue type
```
Comment (jsvigneshkanna) [inline .github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml:27]: replace the whitespace with hypen

**Review by jsvigneshkanna — CHANGES_REQUESTED** [review]
Please replace the whitespace in id value to hypen 
image

**Comment by jsvigneshkanna** [comment]
image
Please replace the whitespace in id value to hypen
@biohazard2117

File: .github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml (hunk 1 above)
Comment (biohazard2117) [reply .github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml:6]: image
Like this ?

Comment (jsvigneshkanna) [reply .github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml:6]: @biohazard2117 like this id: Bug-Title

Comment (biohazard2117) [reply .github/ISSUE_TEMPLATE/ISSUE-TEMPLATE.yml:6]: image

Like this ? @jsvigneshkanna

**Comment by jsvigneshkanna** [comment]
@biohazard2117 Thank you for contribution.

- Please dont forget to star this repo
- Check the issues tab and try to solve similar ones

**Pull request merged**